### DISCUSSION TITLE
Pisugar 3 Battery - No Battery %

### DISCUSSION THREAD

**Discussion by tylergates-dev** [jayofelony/pwnagotchi#282] [Q&A] [answered]
I'm currently stuck trying to get the PiSugar 3 battery to display a percentage through the plugins. It seems to only support PiSugar 2. Do you have any idea when PiSugar 3 will be able to display the battery percentage? I tried following this GitHub guide, but it isn't working for some reason. Any insights on whether JayoFelony has plans to add PiSugar 3 compatibility?

**Comment by wpa-2** [comment, marked as answer]
> I'm currently stuck trying to get the PiSugar 3 battery to display a percentage through the plugins. It seems to only support PiSugar 2. Do you have any idea when PiSugar 3 will be able to display the battery percentage? I tried following this GitHub guide, but it isn't working for some reason. Any insights on whether JayoFelony has plans to add PiSugar 3 compatibility?

It's already included and works have you read the wiki?.

**Reply by tylergates-dev** [reply]
Yes. I must have just missed it. Apologies for the bother
